util/reggen/ip_block.py

```diff
@@ -622,3 +623,34 @@ def check_cm_annotations(self, rtl_names: Dict[str, List[Tuple[str, int]]],
         return CounterMeasure.check_annotation_list(self.name, hjson_path,
                                                     rtl_names,
                                                     self.countermeasures)
+
+    def check_regwens(self):
+        """Checks all regwens are used in at least one other CSR
+
+        This relies on the regwen having the string "REGWEN" in its name.
+        The uses should be in the "regwen" field of a CSR.
+        """
+        status = True
+        log.info(f"Checking regwens for IP {self.name}")
+        for rb in self.reg_blocks.values():
+            rb_name = rb.name if rb.name else "default"
+            log.info(f"Register block: {rb_name}")
+            regwen_names: List[str] = [reg.name for reg in rb.registers
+                                       if "REGWEN" in reg.name]
+            unused_regwens: List[str] = []
+            for regwen in regwen_names:
+                regwen_users = []
```

Comment:
I think this can just be a boolean, can't it? And we can break after finding a regwen user and condition the later loops on the boolean being false?

Reply:
My mistake, I wanted this to be a list to display the regwen uses. This may help diagnose cases where there are multiple uses, but perhaps some are missing. It can be useful in countermeasure reviews. I used this while developing but mistakenly dropped it. Now with -v all uses of each regwen are displayed.